testing/src/main/java/io/grpc/testing/TestUtils.java

```diff
@@ -92,6 +93,23 @@ public static ServerInterceptor recordServerCallInterceptor(
   }
 
   /**
+   * Capture the context seen by the server.  Useful for testing context propagation.
+   */
+  public static ServerInterceptor recordContextInterceptor(
```

Comment:
Let's put this in the one test that uses it. Users shouldn't need this, and it is visible to them here.